src/assets/style.css

```diff
@@ -0,0 +1,278 @@
+  line-height: 88px;
+  position: absolute;
+  left: 88px;
+  top: 211px;
+  z-index: 2;
+}
+
+.trello {
+  height: 56px;
+  width: 131px;
+  border-radius: 28px;
+  background-color: #10bcfc;
+  position: absolute;
+  left: 1163px;
+  top: 64px;
+  display: flex;
+  justify-content: center;
+  align-content: center;
+}
+
+.trello-logo {
+  height: 26px;
+  width: 80px;
+  align-self: center;
+}
+
+.wizard {
+  top: 0px;
+  left: 0px;
+  position: absolute;
+  width: 100%;
+  text-align: center;
+}
+
+.wizard--progress-bar {
+  height: 6px;
+  width: 100%;
+  opacity: 0.16;
+  background-color: #10fc8a;
+  position: absolute;
+  top: 0px;
+  left: 0px;
+}
+
+.wizard--progress-bar--step-1 {
+  width: 33%;
+  opacity: 1;
+}
+
+.wizard--progress-bar--step-2 {
+  width: 66%;
+  opacity: 1;
+}
+
+.wizard--progress-bar--step-3 {
+  width: 100%;
+  opacity: 1;
+}
+
+.wizard--title {
+  width: 100%;
+  color: #211952;
+  font-family: Rubik;
+  font-size: 32px;
+  line-height: 48px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.wizard--text {
+  width: 100%;
+  opacity: 0.56;
+  color: #211952;
+  font-family: Rubik;
+  font-size: 16px;
+  letter-spacing: 0.4px;
+  line-height: 24px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.wizard--text-board {
+  color: #211952;
+  font-size: 18px;
+  letter-spacing: 0.4px;
+  line-height: 26px;
+  margin-top: auto;
+  margin-bottom: auto;
+}
+
+.wizard--text-list {
+  color: rgba(33,25,82,0.56);
+  font-size: 13px;
+  line-height: 1px;
+  margin-left: 5px;
+}
+
+.wizard--text-board-container {
+  margin-left: 0px;
+  margin-bottom: 0px;
+  text-align: left;
+}
+
+.wizard--container {
+  width: 30%;
+  margin: auto;
+  text-align: center;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+.wizard--container-inner {
+  width: 100%;
+}
+
+.wizard--container-center {
+  width: 100%;
+  justify-content: center;
+}
+
+.wizard--board {
+  height: 84px;
+  width: 162px;
+  border: 1px solid rgba(33,25,82,0.24);
+  border-radius: 8px;
+  background-color: #ffffff;
+  box-shadow: 0 9px 11px 0 rgba(85,57,255,0.06);
+  margin-top: 18px;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  text-align: center;
+  overflow: auto;
+}
+
+.wizard--board-selected {
+  background-color: #ffd6ea;
+}
+
+.wizard--button-container {
+  width: 100%;
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: row;
+  justify-content: center;
+  align-content: center;
+  align-items: center;
+  margin-top: 10px;
+}
+
+.button {
+  height: 56px;
+  width: 200px;
+  border: 1px solid #211952;
+  background-color: #ffffff;
+  margin: 5px;
+  color: #211952;
+  font-family: Rubik;
+  font-size: 20px;
+  font-weight: 500;
+  letter-spacing: 0.4px;
+  line-height: 24px;
+}
+
+.button-save {
+  background-color: #fcea10;
+  border: none;
+}
+
+.wizard--container-list {
+  margin-top: 10px;
+  height: 484px;
+  width: 60%;
+  border: 1px solid rgba(255,214,234,0.57);
```

Comment:
Expected single space after "," in a single-line function   function-comma-space-after<br>Unexpected leading zero                                     number-leading-zero